libs/community/langchain_community/document_loaders/url_playwright.py

```diff
@@ -80,6 +81,15 @@ def evaluate(self, page: "Page", browser: "Browser", response: "Response") -> st
             for element in elements:
                 if element.is_visible():
                     element.evaluate("element => element.remove()")
+        
+        selectors = ','.join(self.keep_only_selectors) or 'body'
```

Comment:
Will this technically miss out on things like `<head>` or `<meta>`?

Reply:
`<head>` and `<meta>` stays in the document because I only change body and not the head, so unstructured can partition these two tags even if he will not extract some informations from it (I think)